src/lib/modules/Items/backoffice/ItemListEntry.js

```diff
@@ -75,25 +75,29 @@ export class ItemListEntry extends Component {
             <ItemIcon /> {item.metadata.barcode}
           </Item.Header>
           <Grid columns={2}>
-            <Grid.Column computer={6} largeScreen={6}>
+            <Grid.Column computer={5} largeScreen={5}>
               <Item.Meta className="metadata-fields">
                 <Header as="h5">
                   <LiteratureTitle
                     title={item.metadata.document.title}
                     truncateLines={1}
                   />
                 </Header>
-                <List>
+                <List className="metadata-sharp">
                   <List.Item>
                     <List.Content>
-                      {item.metadata.internal_location.location.name} -{' '}
-                      {item.metadata.internal_location.name}
+                    <div class="ui label">
```

Comment:
in this way you are using the pure semantic ui, which is not what we want to do, since we have a library which provides the components for react. Please check react semantic ui library documentation for more details.